src/Whoops/Run.php

```diff
@@ -303,37 +303,39 @@ public function handleException($exception)
         // we might want to send it straight away to the client,
         // or return it silently.
         $this->system->startOutputBuffering();
-
-        // Just in case there are no handlers:
-        $handlerResponse = null;
-        $handlerContentType = null;
-
-        foreach ($this->handlerQueue as $handler) {
-            $handler->setRun($this);
-            $handler->setInspector($inspector);
-            $handler->setException($exception);
-
-            // The HandlerInterface does not require an Exception passed to handle()
-            // and neither of our bundled handlers use it.
-            // However, 3rd party handlers may have already relied on this parameter,
-            // and removing it would be possibly breaking for users.
-            $handlerResponse = $handler->handle($exception);
-
-            // Collect the content type for possible sending in the headers.
-            $handlerContentType = method_exists($handler, 'contentType') ? $handler->contentType() : null;
-
-            if (in_array($handlerResponse, [Handler::LAST_HANDLER, Handler::QUIT])) {
-                // The Handler has handled the exception in some way, and
-                // wishes to quit execution (Handler::QUIT), or skip any
-                // other handlers (Handler::LAST_HANDLER). If $this->allowQuit
-                // is false, Handler::QUIT behaves like Handler::LAST_HANDLER
-                break;
+        
+        try {
+            // Just in case there are no handlers:
+            $handlerResponse = null;
```

Comment:
These assignments could be moved before the try/catch

Reply:
You are so right, my bad, preparing to update PR